src/Psd/PsdCompilerMacros.h

```diff
@@ -144,3 +144,11 @@
 	#define static_assert(condition, message)			typedef char PSD_JOIN(static_assert_impl_, __LINE__)[(condition) ? 1 : -1]
 	#define nullptr										NULL
 #endif
+
+#define PSD_STATIC_ASSERT_MSG(condition, message)		static_assert((condition), message)

```

Comment:
Is PSD_STATIC_ASSERT_MSG used anywhere?

Reply:
not using now
often, static_assert since C++11 and C++17 confused
(actually at sample code)
if possible, static_assert with message  should be specified
for example STATIC_ASSERT_MSG